tests/unit/multiclass/BaggingMachine_unittest.cc

```diff
@@ -27,11 +30,168 @@
 using namespace shogun;
 using ::testing::Return;
 
+class BaggingMachine : public ::testing::Test
+{
+public:
+	CDenseFeatures<float64_t>* features_test;
+	CDenseFeatures<float64_t>* features_train;
+	CMulticlassLabels* labels_train;
+
+	SGVector<bool> ft;
+	virtual void SetUp()
+	{
+		sg_rand->set_seed(1);
+		generate_toy_data_weather();
+	}
+
+	virtual void TearDown()
+	{
+		SG_UNREF(features_train);
+		SG_UNREF(features_test);
+		SG_UNREF(labels_train);
+	}
+
+	void generate_toy_data_weather()
```

Comment:
Is it appropriate to create a generate_test_data header and cpp file just for data generation? Should I place this in the tests/unit/utils folder? Sorry for asking a lot for this point, I just want to avoid re-changing it later.